toc.hbs.md

```diff
@@ -601,9 +601,9 @@ docs.vmware.com is built.
             - [Troubleshooting Rego Files](scst-scan/troubleshoot-rego.hbs.md)
         - [Supply Chain Security Tools - Scan 2.0 (beta)](scst-scan/app-scanning-beta.hbs.md)
             - [Getting Started](scst-scan/getting-started.hbs.md)
-                - [Installation](scst-scan/install-app-scanning.hbs.md)
+                - [Installing SCST - Scan 2.0](scst-scan/install-app-scanning.hbs.md)
                 - [Using App-Scanning in Supply Chains](scst-scan/integrate-app-scanning.hbs.md)
-                - [Bring your own scanner](scst-scan/bring-your-own-scanner.hbs.md)
+                - [Using App-Scanning with other Scanners in Supply Chains](scst-scan/bring-your-own-scanner.hbs.md)
```

Comment:
App-Scanning here isn't consistent with Scan 2.0 throughout the rest of the TOC.

Reply:
Er.  Nevermind.  It is above too.